workspaces/sandbox/app-config.yaml

```diff
@@ -6,20 +6,11 @@ organization:
   name: Red Hat
 
 sandbox:
+  signupAPI: ${SIGNUP_API}
+  kubeAPI: ${KUBE_API}
```

Comment:
just a reminder - are you going to open a PR in sandbox-sre also and adjust the config accordingly ?

Reply:
Sure